screens/LoadingScreen/LoadingScreen.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react'
+import Spinner from 'react-native-loading-spinner-overlay'
+import tailwind from 'tailwind-rn'
+import { View } from '../../components/Themed'
+import { translate } from '../../translations'
+import { StyleSheet } from 'react-native'
+
+interface LoadingScreenProps {
+  message?: string
+}
```

Comment:
don't declare props interface if it's just one variable, more cognitive load to read and understand code in long term maintainability

Reply:
@fuxingloh it fails the TS validation due to having infer `any` type if we remove the interface.

Reply:
```ts
export default function LoadingScreen (props: {message?: string}): JSX.Element {
```

Reply:
Wouldn't it look better with an interface? Especially if we want to add more params in the future?

Reply:
Example on why we would want to have an interface even for one variable. - https://github.com/DeFiCh/wallet/blob/818be79b6cc587df1ea6fb7b810bd78680237e2c/screens/ErrorBoundary/ErrorBoundary.tsx#L24

We declare an interface for the return type and method return types (`getDerivedStateFromError`) and in props.

Reply:
> Wouldn't it look better with an interface? Especially if we want to add more params in the future?

Not really, best not to do early premature optimization, especially when we have no idea of how it will evolve.

Reply:
> Example on why we would want to have an interface even for one variable. -
> 
> https://github.com/DeFiCh/wallet/blob/818be79b6cc587df1ea6fb7b810bd78680237e2c/screens/ErrorBoundary/ErrorBoundary.tsx#L24
> 
> We declare an interface for the return type and method return types (`getDerivedStateFromError`) and in props.

I think more and more conventions are written in use hooks now, might not be relevant.

Reply:
Ok I will update. 

For `getDerivedStateFromError` there's no hooks for it yet. See https://reactjs.org/docs/hooks-faq.html#do-hooks-cover-all-use-cases-for-classes